lib/arel_extensions.rb

```diff
@@ -77,7 +77,13 @@ class Arel::Nodes::Case
 require 'arel_extensions/nodes/cast'
 require 'arel_extensions/nodes/json'
 
-
+if defined?(Arel::Visitors::SQLServer)
```

Comment:
Please, put comments documenting dark magic.  Commit messages are not enough.  Especially not for obscure stuff.

Reply:
Done.